site/content/en/docs/formats/synthia.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'SYNTHIA'
+linkTitle: 'SYNTHIA'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original SYNTHIA dataset is available
+[here](https://synthia-dataset.net).
```

Comment:
So while the website is called "The SYNTHIA dataset", there seems to not actually be a single dataset called SYNTHIA there. Instead, there are several datasets like SYNTHIA-AL, SYNTHIA-SF, etc. Have you checked that they all have the same format?

Reply:
Yes, I checked. There is a difference only with SYNTHIA-AL. It contains 3D detection, and folder names differ for segmentation. I think that for the purposes of DL Workbench this will not be a problem.

Reply:
So am I understanding correctly that SYNTHIA-AL is not supported? If so, that should be stated in the documentation.

Reply:
Added this to documentation